### PR TITLE
Update CI/CD diagrams

### LINKED ISSUE #1324: Update images for the CI/CD docs
The images on the CI/CD docs need updating to improve the background, layout, and resolution. 

Related PR: https://github.com/elastic/observability-docs/pull/1266

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Fixes #1324 

Updates CI/CD diagrams to improve background, layout, and resolution.

### CODE REVIEW

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ OpenTelemetry.
 Using the APM Server, connect all your OpenTelemetry native CI/CD tools directly to Elastic Observability.
 
 .Architecture of CI/CD Observability with Elastic Observability
-image::images/simple-arch-observability.png[Simple architecture of CI/CD Observability]
+image::images/simple-arch-observability.png[Simple architecture of CI/CD Observability with Elastic Observability]
```
Comment (ollyhowell) [inline docs/en/observability/ci-cd-observability.asciidoc:20]:
```suggestion
image::images/simple-arch-observability.png[Simple architecture of CI/CD with Elastic Observability]
```
Remove double "Observability"?

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 2]
```diff
@@ -28,7 +28,7 @@ ephemeral tools like the otel-cli.
 Elastic Observability.
 
 .Advanced architecture of CI/CD Observability with Elastic Observability
-image::images/advanced-arch-observability.png[Advanced architecture of CI/CD Observability]
+image::images/advanced-arch-observability.png[Advanced architecture of CI/CD Observability with Elastic Observability]
```
Comment (ollyhowell) [inline docs/en/observability/ci-cd-observability.asciidoc:31]:
```suggestion
image::images/advanced-arch-observability.png[Advanced architecture of CI/CD with Elastic Observability]
```

**Review by ollyhowell — APPROVED** [review]
Diagrams look so much cleaner! Thanks for fixing these, left a minor comment about image caption

File: docs/en/observability/ci-cd-observability.asciidoc (hunk 1 above)
Comment (colleenmcginnis) [reply docs/en/observability/ci-cd-observability.asciidoc:20]: I just updated `alt` text to match the existing visible caption above (`.Architecture of CI/CD Observability with Elastic Observability`). I can remove from both the caption and the `alt` text.

**Review by cyrille-leclerc — APPROVED** [review]
THANK YOU very much

**Comment by cyrille-leclerc** [comment]
> Diagrams look so much cleaner! Thanks for fixing these, left a minor comment about image caption

@ollyhowell I like very much my diagrams as well 😄 . Maybe a different audience.

**Pull request merged**